samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -71,6 +71,8 @@ export interface HomeScreenProps {
     teamsToken?: string;
     /* @conditional-compile-remove(teams-identity-support) */
     teamsId?: string;
+    /* @conditional-compile-remove(teams-adhoc-call) */
+    teamsIds?: string[];
```

Comment:
can we call this `outboundTeamsUsers` or something so we know it is the identifiers that we are calling?